docs/en/ingest-management/elastic-agent/example-kubernetes-fleet-managed-agent-helm.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[example-kubernetes-fleet-managed-agent-helm]]
+= Example: Install {fleet}-managed {agent} on Kubernetes using Helm
+
+preview::[]
+
+This example demonstrates how to install {fleet}-managed {agent} on a Kubernetes system using a Helm chart, gather Kubernetes metrics and send them to an {es} cluster in {ecloud}, and then view visualizations of those metrics in {kib}.
+
+For an overview of the {agent} Helm chart and its benefits, refer to <<install-on-kubernetes-using-helm>>.
+
+This guide will take you through the following steps:
+
+* <<agent-fleet-managed-helm-example-install-agent>>
+* <<agent-fleet-managed-helm-example-install-integration>>
+* <<agent-fleet-managed-helm-example-tidy-up>>
+
+
+[discrete]
+[[agent-fleet-managed-helm-example-prereqs]]
+=== Prerequisites
+
+To get started, you need:
+
+* A local install of the link:https://helm.sh/[Helm] Kubernetes package manager.
+* An link:{ess-trial}[{ecloud}] hosted {es} cluster on version 8.16 or higher.
+* An active Kubernetes cluster.
+* A local clone of the link:https://github.com/elastic/elastic-agent/tree/8.16[elastic/elastic-agent] GitHub repository. Make sure to use the `8.16` branch to ensure that {agent} has full compatibility with the Helm chart.
```

Comment:
```suggestion
* A local clone of the link:https://github.com/elastic/elastic-agent/tree/{branch}[elastic/elastic-agent] GitHub repository. 
```

Reply:
Try the `{branch}` variable rather than hardcoding to future proof docs. I think it _should_ work, but worth testing.  

If the `{branch}` variable works, this might be a place where we can remove the `8.16` note.

Reply:
Great idea! Normally I would use a variable for this, but:

 - It looks like the `8.16` appears only here, at lines 24 and 27.
 - I expect we'll need to revise this part for 8.17 anyway, since it's planned to move the Helm chart out of the elastic-agent repo.

Reply:
Completely agree. Thanks for explaining.